`cargo doc` fails to document `ruma_events`

I tried to run `cargo doc` on a project depending on ruma 0.9.2 and the matrix rust sdk. However, it failed to finish with the following error message:

```
error[E0432]: unresolved import `ruma_html`
 --> /usr/local/share/rust/registry/src/github.com-1ecc6299db9ec823/ruma-events-0.27.7/src/room/message/without_relation.rs:4:5
  |
4 | use ruma_html::{sanitize_html, RemoveReplyFallback};
  |     ^^^^^^^^^ use of undeclared crate or module `ruma_html`

For more information about this error, try `rustc --explain E0432`.
error: could not document `ruma-events`
```

--- a/crates/ruma-events/src/doc/rich_reply.md
+++ b/crates/ruma-events/src/doc/rich_reply.md
@@ -1,8 +1,7 @@
 <!-- Keep this comment so the content is always included as a new paragraph -->
 This function requires an [`OriginalRoomMessageEvent`] since it creates a permalink to
 the previous message, for which the room ID is required. If you want to reply to an
-[`OriginalSyncRoomMessageEvent`], you have to convert it first by calling
-[`.into_full_event()`][crate::OriginalSyncMessageLikeEvent::into_full_event].
+[`OriginalSyncRoomMessageEvent`], you have to convert it first by calling [`.into_full_event()`].
 
 If the message was edited, the previous message should be the original message that was edited,
 with the content of its replacement, to allow the fallback to be accurate at the time it is added.
@@ -11,4 +10,9 @@ It is recommended to enable the `html` feature when using this method as this wi
 clean up nested [rich reply fallbacks] in chains of replies. This uses [`sanitize_html()`]
 internally, with [`RemoveReplyFallback::Yes`].
 
+[`OriginalRoomMessageEvent`]: crate::room::message::OriginalRoomMessageEvent
+[`OriginalSyncRoomMessageEvent`]: crate::room::message::OriginalSyncRoomMessageEvent
+[`.into_full_event()`]: crate::OriginalSyncMessageLikeEvent::into_full_event
 [rich reply fallbacks]: https://spec.matrix.org/latest/client-server-api/#fallbacks-for-rich-replies
+[`sanitize_html()`]: ruma_html::sanitize_html
+[`RemoveReplyFallback::Yes`]: ruma_html::RemoveReplyFallback::Yes

--- a/crates/ruma-events/src/room/message/without_relation.rs
+++ b/crates/ruma-events/src/room/message/without_relation.rs
@@ -1,11 +1,7 @@
 use as_variant::as_variant;
 use ruma_common::{serde::Raw, OwnedEventId, OwnedUserId, RoomId, UserId};
-#[cfg(doc)]
-use ruma_html::{sanitize_html, RemoveReplyFallback};
 use serde::{Deserialize, Serialize};
 
-#[cfg(doc)]
-use super::OriginalSyncRoomMessageEvent;
 use super::{
     AddMentions, ForwardThread, MessageType, OriginalRoomMessageEvent, Relation,
     RoomMessageEventContent,

